tools/rtpsrelay/RtpsRelay.cpp

```diff
@@ -259,9 +260,12 @@ int ACE_TMAIN(int argc, ACE_TCHAR* argv[])
   ACE_INET_Addr spdp(rtps_discovery->get_spdp_port(application_domain, application_participant_id), "127.0.0.1");
   ACE_INET_Addr sedp(rtps_discovery->get_sedp_port(application_domain, application_participant_id), "127.0.0.1");
 
-  SpdpHandler spdp_vertical_handler(reactor, association_table, lifespan, application_participant_id, spdp);
-  SedpHandler sedp_vertical_handler(reactor, association_table, lifespan, application_participant_id, sedp);
-  DataHandler data_vertical_handler(reactor, association_table, lifespan, application_participant_id);
+  OpenDDS::Security::SecurityConfig_rch conf = TheSecurityRegistry->default_config();
```

Comment:
Need to cover builds that are have security disabled at compile time using `OPENDDS_SECURITY` preprocessor macro